velox/vector/fuzzer/ConstrainedVectorGenerator.cpp

```diff
@@ -137,7 +58,10 @@ VectorPtr ConstrainedVectorGenerator::generateFlat(
       writer.commitNull();
     } else {
       VELOX_DYNAMIC_TYPE_DISPATCH(
-          writeOne, type->kind(), variant, writer.current());
+          generator_spec_utils::writeOne,
```

Comment:
nit: It looks like writeOne is not relevant to generator_spec_utils, so could we move it out of this namespace?